### PR TITLE
525 bootstrap element list group

### PR DESCRIPTION
**Pull request by Delnote**
525 bootstrap element list group

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/CustomContentTests.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+public class CustomContentTests extends TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    String link1 = "https://jdi-testing.github.io/jdi-light/index.html";
+    String Link2 = "https://github.com/jdi-testing";
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/CustomContentTests.java]: link2

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ListGroupAssert.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+public class ListGroupAssert extends UISelectAssert<ListGroupAssert, ListGroup>
+        implements SelectedAssert<ListGroupAssert> {
+
+    @JDIAction("Assert that '{name}' is selected")
+    public ListGroupAssert selected() {
+        jdiAssert(getIsSelected(), is("selected"));
+        return this;
+    }
+    @JDIAction("Assert that '{name}' is not selected")
```
Comment (mf-code) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ListGroupAssert.java]: need empty string between methods

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/WithBadgesTests.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                .tag(is("span"));
+        listGroupWithBadges.badge.get(3).is()
+                .text("1")
+                .css("font-size", is("10.5px"))
+                .hasClass("badge badge-primary badge-pill")
+                .tag(is("span"));
+    }
+
+
+
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/WithBadgesTests.java]: three empty strings presented

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/CustomContentTests.java (hunk 1 above)
Comment (Delnote) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/CustomContentTests.java]: Fixed.

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ListGroupAssert.java (hunk 2 above)
Comment (Delnote) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ListGroupAssert.java]: Fixed.

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/WithBadgesTests.java (hunk 3 above)
Comment (Delnote) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/WithBadgesTests.java]: Fixed.

**Comment by mf-code** [comment]
image
please fix codacy issues except of package naming

**Pull request merged**